src/cnc/flavors/aws/ecs/1/provision/partials/collection/lambda/config_alb.tf.j2

```diff
@@ -0,0 +1,90 @@
+{# Use lb_rules.priority and sorted_services globally initialized #}
+
+    resource "aws_alb_target_group" "{{ service.instance_name }}_tg" {
+      name        = "{{ service.instance_name[:29] }}-tg"
+      target_type = "lambda"
+      vpc_id      = {{ env_collection.vpc_resource_address }}.id
+    }
+
+    resource "aws_alb_target_group_attachment" "{{ service.instance_name }}_tg_attachment" {
+      target_group_arn = aws_alb_target_group.{{ service.instance_name }}_tg.arn
+      target_id        = aws_lambda_function.{{ service.instance_name }}.arn
+    }
+
+    resource "aws_lb_listener_rule" "{{ env_collection.instance_name }}_{{ lb_rules.priority }}" {
+      listener_arn = aws_alb_listener.{{ env_collection.instance_name }}_http.arn
```

Comment:
I think this is the issue with the routing, this should be attached to the https listener instead of the http listener ^

Reply:
solved